### PR TITLE
Add guidance for designing for limited english

### PR DESCRIPTION
**Pull request by sshukor**

### CODE REVIEW

**Review by sshukor — COMMENTED** [review]
Image added to new folder

File: apps/docs/src/common/pages/content-style-guide/designing-for-limited-english.tsx [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+import { Helmet } from 'react-helmet-async';
+import { PageProps } from '@not-govuk/app-composer';
+import { A } from '@not-govuk/components';
+import { menu } from '../accessibility'
+
+export const title = 'Limited English';
+const description = 'Designing for people with limited English';
+export const section = 'Accessibility';
+
+const applyImageInline = require('../../../../assets/images/content-style-guide/Biometrics sign.png');
```
Comment (daniel-ac-martin) [inline apps/docs/src/common/pages/content-style-guide/designing-for-limited-english.tsx:11]:
```suggestion
const biometricsSign = require('../../../../assets/images/content-style-guide/Biometrics sign.png');
```

File: apps/docs/src/common/pages/content-style-guide/designing-for-limited-english.tsx [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+
+      <h2 id="avoid-all-contractions">Avoid all contractions</h2>
+      <p>Avoid all contractions, including positive contractions like 'we'll', 'you've' or 'they're', if your users have limited English. Research suggests that contractions make content difficult to understand for people with limited fluency. See general guidance on contractions in the <A href="https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style">GOV.UK style guide</A>. </p>
+       
+      
+
+      <h2 id="consider-images-or-videos">Consider images or video</h2>
+      <p>In some cases, an image, diagram or video can communicate information more clearly. Research at UK arrival centres found that some people had difficulty finding their way using the text-only signage. To address this issue, designers produced a new signage system using text supported by images.</p>
+       
+      <img src={applyImageInline} alt="Sign showing camera and fingerprint icons to explain in different languages how biometric information is taken." className="image-examples" />
```
Comment (daniel-ac-martin) [inline apps/docs/src/common/pages/content-style-guide/designing-for-limited-english.tsx:194]:
```suggestion
      <img src={biometricsSign} alt="Sign showing camera and fingerprint icons to explain in different languages how biometric information is taken." className="image-examples" />
```

File: apps/docs/src/common/pages/content-style-guide/designing-for-limited-english.tsx [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+  <h2 id="avoid-idioms">Avoid idioms</h2>
+        <p>The English language has many idioms. We use them to make our services more conversational. Common expressions like 'by the book', 'red tape' or 'touch base' are not always clear to people with limited English and do not always translate well using translation tools like Google Translate or Bing Translator. </p> 
+        <p>Designers working on the passport application service found that users with limited English did not recognise 'wear and tear' as an expression to mean damage. Instead, they translated these words individually and mistakenly thought the statement did not apply to their passports as they were not torn.  </p>
+
+ <h2 id="avoid-phrasal-verbs">Avoid phrasal verbs</h2>
+       <p>A phrasal verb is where you combine a verb with other words to make it mean something different, such as ‘carry out’, ‘get over’ or ‘look after’. English is full of them. But their meaning is not always clear until you have learned them.  </p>
+       <p>There is nearly always a one-word alternative to a phrasal verb that is easier to translate.  </p>
+
+       <div class="govuk-grid-row">
+  <div class="govuk-grid-column-one-half">
```
Comment (daniel-ac-martin) [inline apps/docs/src/common/pages/content-style-guide/designing-for-limited-english.tsx:100]:
```suggestion
       <div className="govuk-grid-row">
  <div className="govuk-grid-column-one-half">
```

**Review by daniel-ac-martin — APPROVED** [review]

**Pull request merged**